src/main/java/com/basho/riak/client/core/query/search/YokozunaIndex.java

```diff
@@ -24,7 +24,8 @@
 {
     private final String name;
     private final String schema;
-    
+    private volatile Integer nVal;
```

Comment:
yaRly!  We set it with withNVal() a few lines down, and not in the constructor. We typically mark any fields non-final fields as volatile in these "core" datatype objects, to make them thread-safer with all the fun setup.  I also didn't want to add two more constructors :).